worlds/soe/__init__.py

```diff
@@ -4,18 +4,21 @@
 import threading
 import typing
 
+# from . import pyevermizer  # as part of the source tree
+import pyevermizer  # from package
+
 import settings
+from BaseClasses import Item, ItemClassification, Location, LocationProgressType, Region, Tutorial
+from Utils import output_path
 from worlds.AutoWorld import WebWorld, World
 from worlds.generic.Rules import add_item_rule, set_rule
-from BaseClasses import Entrance, Item, ItemClassification, Location, LocationProgressType, Region, Tutorial
-from Utils import output_path
+from . import logic  # load logic mixin
```

Comment:
.. but it's not an AP logicmixin anymore?

Reply:
~~that's why i don't like Ctrl+Alt+O~~